pkgs/servers/mastodon/update.sh

```diff
@@ -77,7 +77,8 @@ trap cleanup EXIT
 
 echo "Fetching source code $REVISION"
 JSON=$(nix-prefetch-github "$OWNER" "$REPO" --rev "$REVISION" 2> $WORK_DIR/nix-prefetch-git.out)
-HASH=$(echo "$JSON" | jq -r .hash)
+HASH="$(echo "$JSON" | jq -r .sha256)"
+HASH="$(nix hash to-sri --type sha256 "$HASH")"
```

Comment:
This breaks the update script if you are on nixos-unstable. Going to revert it.

Reply:
Weird, I thought I tested this. Does this help? https://github.com/NixOS/nixpkgs/pull/257810